biogeochem/EDPhysiologyMod.F90

```diff
@@ -1332,33 +1332,34 @@ subroutine CWD_Input( currentSite, currentPatch)
          currentPatch%cwd_BG_in(c) = currentPatch%cwd_BG_in(c) + (struct_c + sapw_c) * & 
               SF_val_CWD_frac(c) * dead_n * (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft))
          
-         ! Send AGB component of boles from non direct-logging activities to AGB litter pool
+         ! Send AGB component of boles from both direct and non-direct logging activities to AGB litter pool
```

Comment:
How adding some text that indicates non-exported portion of direct harvesting:

```
! Send AGB component of boles from logging activities into the litter.  
! This includes fluxes from indirect modes of death, as well as the 
! non-exported boles due to direct harvesting.
```

Note, my terminology may be wrong.  I'm assuming that a bole is the above ground portion of a trunk, and that term is appropriate for natural vegetation as well as wood products.

Reply:
Done. please check again.